lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -372,13 +366,9 @@ export default function getRecipeInterface(
                     res
                 );
             } catch (err) {
-                if (
-                    (err.type === STError.UNAUTHORISED && err.payload.clearCookies) ||
-                    err.type === STError.TOKEN_THEFT_DETECTED
-                ) {
-                    logDebugMessage(
-                        "refreshSession: Cookies will get cleared because of UNAUTHORISED or TOKEN_THEFT_DETECTED response"
-                    );
+                if (err.type === STError.TOKEN_THEFT_DETECTED) {
+                    logDebugMessage("refreshSession: Clearing cookies because of TOKEN_THEFT_DETECTED response");
+                    clearSessionFromCookie(config, res);
```

Comment:
token theft detection also clears cookies by default right? (see the change you made..?)